Samples/MobileBuyIntegration/MobileBuyIntegration/ProductView.swift

```diff
@@ -0,0 +1,290 @@
+/*
+MIT License
+
+Copyright 2023 - Present, Shopify Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+import Buy
+import UIKit
+import SwiftUI
+import ShopifyCheckoutSheetKit
+
+struct ProductView: View {
+    // MARK: Properties
+    @State private var product: Storefront.Product
+    @State private var handle: String?
+    @State private var loading = false
+	@State private var imageLoaded: Bool = false
```

Comment:
Can't tell if it's Github or the indentation is actually a bit off?

Reply:
Yeah it's actually off. https://github.com/Shopify/checkout-sheet-kit-swift/pull/247 will fix it